wes_service/cwl_runner.py

```diff
@@ -178,12 +181,31 @@ def ListWorkflows(self):
             "next_page_token": ""
         }
 
-    def RunWorkflow(self, body):
-        # FIXME Add error responses #16
-        if body["workflow_type"] == "CWL" and body["workflow_type_version"] != "v1.0":
+    def RunWorkflow(self):
+        tempdir = tempfile.mkdtemp()
+        body = {}
+        for k, ls in connexion.request.files.iterlists():
+            for v in ls:
+                if k == "workflow_descriptor":
+                    filename = secure_filename(v.filename)
+                    v.save(os.path.join(tempdir, filename))
+                elif k in ("workflow_params", "tags", "workflow_engine_parameters"):
+                    body[k] = json.loads(v.read())
+                else:
+                    body[k] = v.read()
+
+        body["workflow_url"] = "file:///%s/%s" % (tempdir, body["workflow_url"])
+        index = body["workflow_url"].find("http")
+        if index > 0:
+            body["workflow_url"] = body["workflow_url"][index:]
+
+        if body['workflow_type'] != "CWL" or \
```

Comment:
Just a nit, but this check might be better at the beginning of the function the way it was.